install_config/web_console_customization.adoc

```diff
@@ -19,6 +19,54 @@ using extensions, which let you run scripts and load custom stylesheets when the
 web console loads. You can change the look and feel of nearly any aspect of the
 user interface in this way.
 
+== Configuring Web Console Customizations with Ansible
```

Comment:
I would make this the last section, on this document, not the first. 

In short the flow should be to explain what can be configurable, and then to explain how it can be configured with Ansible.